CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
```

Comment:
please rebase onto latest dev

Reply:
Rebased with upstream/dev branch and did not notice any gaps with upstream. Are you suggesting that this note should be under "### Changed"  instead of "### Fixed"?